### DISCUSSION TITLE
2 missing types

### DISCUSSION THREAD

**Discussion by charlesrbasin** [code4recovery/12-step-meeting-list#1198] [Q&A]
Meetings with 2 types do not show the missing types on the detail page. When we look at the meeting it has the blue check mark. The missing types are LGBTQIA+ and Parents of Alcoholics. We do not see a problem with any of the other types.
[https://staging7.alanonma.org/meetings/?search=25035](url)
TSML  Version 3.14.19
WP version: 6.3
These 2 types are missing from the dropdown for types as well.
![Edit Meeting “25035 Testing meeting Types AFG” ‹ Test_Development site — WordPress](https://github.com/code4recovery/12-step-meeting-list/assets/47933751/81887248-bfb8-42f0-aa1a-1ce9f2f5575c)
![25035 Testing meeting Types AFG](https://github.com/code4recovery/12-step-meeting-list/assets/47933751/2c239af4-4794-4f29-ba78-a6980f890047)

**Comment by joshreisner** [comment]
looks like a bug - created an issue to track it here: https://github.com/code4recovery/tsml-ui/issues/325

**Reply by joshreisner** [reply]
version `1.6.2` is released with a fix 🚀

**Comment by charlesrbasin** [comment]
I just discovered that the types are missing in TSML UI version but not in Legacy UI.

**Reply by joshreisner** [reply]
yep, should be fixed now if/when you switch back to TSML UI. (you may also need to refresh your browser)

**Comment by charlesrbasin** [comment]
I should wait for next release of plugin 3.14.20?

 

Barry Nelson

   ***@***.***> ***@***.***

   <https://alanonma.org/> alanonma.org

**Reply by joshreisner** [reply]
not necessarily, it should be working now

**Comment by charlesrbasin** [comment]
All set. Thanks.
